tests/framework/clients/bucket.go

```diff
@@ -71,6 +73,30 @@ func (b *BucketOperation) CheckOBC(obcName, check string) bool {
 	}
 	logger.Infof("%s resources %v all %s", obcName, resources, check)
 
+	if shouldBeBound {
+		// OBC should be in bound phase as well as existing
+		state, _ := b.k8sh.GetResource("obc", obcName, "--output", "jsonpath={.status.phase}")
+		if state != "Bound" {
```

Comment:
What about using the type `ObjectBucketStatusPhase` with `ObjectBucketStatusPhaseBound` from obc v1alpha1 types?